### DISCUSSION TITLE
I get an error on generate using npx prisma generate

### DISCUSSION THREAD

**Discussion by P3ntest** [prisma/prisma#12475] [Q&A] [answered]
Hey, I am on windows and I just setup a project with prisma and postgresql. I can push to the db without problems, but when I run npx prisma generate I get the following error:

Error: spawn postgresql ENOENT

What am I doing wrong?

**Comment by janpio** [comment, marked as answer]
What does your `schema.prisma` file look like? I think you might have a `generator` block with `provider = "postgresql"`. That should only appear in a `datasource` block though.

**Reply by P3ntest** [reply]
I checked and that actually seemed to be the issue. I must have done that wrong following the tutorial. Thank you so much!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
